examples/viewer.js

```diff
@@ -248,7 +252,7 @@ let dataChannelLatencyCalcMessage = {
 async function startViewer(localView, remoteView, formValues, onStatsReport, remoteMessage) {
     try {
         console.log('[VIEWER] Client id is:', formValues.clientId);
-        viewerButtonPressed = Date.now();
+        viewerButtonPressed = new Date();
```

Comment:
If we use Date.now() we won't have to use .getTime() everywhere

Reply:
Date.now() breaks ISO string conversion we do in DQP metrics. So reverted to using this.